sda-commons-web-autoconfigure/src/main/java/org/sdase/commons/spring/boot/web/client/EnableSdaClients.java

```diff
@@ -68,9 +69,35 @@
  * configured {@code "oidc.client.issuer.uri"}, {@code "oidc.client.id"} and {@code
  * "oidc.client.secret"}. If the current request context contains the {@code Authorization} header,
  * the authentication pass-through will be applied instead.
+ *
+ * <p>The client can be used within the SDA Platform to path through the received Trace-Token header
+ * by adding a configuration:
+ *
+ * <pre>
+ *   <code>{@literal @}FeignClient(
+ *     name = "partnerOds",
+ *     url = "${partnerOds.baseUrl}",
+ *     configuration = {AuthenticationPassThroughClientConfiguration.class})
```

Comment:
Should this be the `SdaTraceTokenClientConfiguration`?
Trace Token support may be added to the list at the beginning of this JavaDoc as well.